### PR TITLE
COVID Statistics Capstone Project Using React & Redux.

### PR DESCRIPTION
**Pull request by HSMathebula**
**Documentation Link: [Loom Vedio](https://www.loom.com/share/c0fd2fb663054839ae513c7875214e93)**
**Home page**

- When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for 
  - example by the category name.
  - Along with the category name, you will display some numeric values per category that come from the API.
  - When the user clicks (or taps) on a category item, the application navigates to the details page.
  
- Details page

  - On the details page, the web app retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.
  
**Testing requirements**

- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.
  - You may need to mock the access to the API so that your tests don't send actual requests.
  - You may need to mock the connection to the Redux Store.
  
**Technical Requirements**

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- You should filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project should be deployed and accessible online (add a link to the online version of your app to your README file).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,52 @@
...
 
-In the project directory, you can run:
+### Built With
+- React
+- Redux
+- JavaScript
 
-### `npm start`
+## Live Demo Links
+'Will go here'
```
Comment (AnselemOdims) [inline README.md]: - [ ] Kindly host your app and include the live link here

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @HSMathebula , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please kindly write unit tests for the DetailPage component and integration tests using React Testing Library
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @HSMathebula, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
- [ ] Current live link not working as expected instead it keep showing the following 👇 
Screenshot (288)
 Kindly fix it and request for another review.

_Do not check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by HSMathebula** [comment]
Hi @elyor-doniyorov, I don't know how to deploy a branch that is not merged. I'm used to deploying using develop branch which is the default branch in my repo.

**Comment by HSMathebula** [comment]
@elyor-doniyorov the link will work when I merge to develop. or im suppose to submit a pull request to merge to the main?

**Comment by elyor-doniyorov** [comment]
> @elyor-doniyorov the link will work when I merge to develop. or im suppose to submit a pull request to merge to the main?

hi @HSMathebula, good question, thanks. You can deploy your feature branch to Netlify by choosing that specific branch in branches section

**Comment by HSMathebula** [comment]
Thank you @elyor-doniyorov

**Review by DammyShittu — APPROVED** [review]
Hi @HSMathebula,

## APPROVED 🟢

✔️ Filter implementation

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**